albo.py

```diff
@@ -24,44 +13,41 @@ def send_telegram_message(text):
     URL = "https://api.telegram.org/bot" + TOKEN + "/sendMessage?&parse_mode=markdown&" + urlencode(params, quote_via=quote_plus)
     r = requests.get(url = URL)
 
+def get_headers(table):
+  headers = []
+  for header in table.find('tr').find_all("td"):
+    headers.append(header.string)
+  return headers
 
-headers = [
-  "Numero",
-  "Data registrazione",
-  "Richiedente",
-  "Oggetto",
-  "Inizio pubblicazione",
-  "Fine pubblicazione",
-]
-
-driver = webdriver.Chrome(options=chrome_options)
-
-driver.get('https://ws1.unict.it/albo/')
+f = open("last_id.txt", "r")
+last_id = int(f.read())
+f.close()
 
-file = open('last_id.txt', 'r')
-last_id = int(file.read())
-file.close()
+page = requests.get("http://albo.unict.it")
 
-id = driver.execute_script("return document.querySelectorAll('.records tr')[1].querySelectorAll('td')[0].innerText")
+if page.status_code != 200:
+  print("Impossibile accedere al sito, riprovare più tardi")
 
-if last_id != id:
-  diff = int(id) - last_id
+soup = BeautifulSoup(page.content, 'html.parser')
 
-  for x in range(diff):
-    data = driver.execute_script("return [...document.querySelectorAll('.records tr')[" + str(x+1) +"].querySelectorAll('td')].map(x => x.querySelector('span').innerText)")
+table = soup.find('div', id='boge')
 
-    content = ""
+new_id = int (table.find('tr').find_next_sibling().td.string)
 
-    for idx in range(len(headers)):
-      print(headers[idx] + ": " + data[idx])
-      content += "*" + headers[idx] + "*: " + data[idx] + "\n"
+#print(new_id)
 
-    print("\n")
-    send_telegram_message(content)
+headers = get_headers(table)
 
-# update id
-file = open("last_id.txt", "w+")
-file.write(id)
-file.close()
+for id in range (last_id + 1, new_id + 1):
+  tr = table.find('td', text=id).parent
+  row = tr.find_all('td')
+  message = ""
+  for i, header in enumerate(headers):
+    #print(row[i].get_text())
+    message = message + "*" + header + "*: " + row[i].span.string + "\n"
```

Comment:
for now this is ok, but I would like to improve the format of the message that we publish into the telegram channel

Reply:
I just sticked to the current format, if something else is preferable please comment below an example of such format